frontend/src/translations/result/br.json

```diff
@@ -420,6 +420,7 @@
   "welcome.how.workflow.main.title": "Como trabalhar",
   "welcome.how.workflow.contrib.title": "Para contribuidores",
   "payment.options.warning.message": "Ativando sua conta você irá receber pagamentos por transferência direta. Se o país onde você possui conta não estiver na lista, você pode ativar sua conta usando o Paypal. No entanto, com o Paypal, as taxas dele serão aplicadas.",
+  "user.confirm.password.error": "As senhas não coincidem",
```

Comment:
As mentioned, you don't need to add an entry here, just one entry at:
https://github.com/worknenjoy/gitpay/blob/master/frontend/src/translations/en.json